packages/api/src/routes/connect.ts

```diff
@@ -211,6 +214,7 @@ router.get(
       },
     });
 
+    await sendUpdatedUserProviderList(token, cxId, userId);
     return res.status(status.OK).send(connectedUser.id);
```

Comment:
Why are we sending a WH request when the cx requests the list of providers? Shouldn't we just return those on the response payload?

Reply:
🤔 The suggested solution in the issue description says WH payload. Did I misunderstand this?

Reply:
Oh, this is the same as the OAuth ones but for Apple. I'm sorry, I thought this was an endpoint for our cxs to get the list of connected providers for a given user. Disregard, please.